src/main/java/org/qualityannotate/quality/sonarqube/client/SonarqubeApiClient.java

```diff
@@ -9,49 +8,84 @@
-     */
+     * @param project
+     *                   Component key. E.g. my_project
+     * @param prId
+     *                   Pull request id. Not available in the community edition. E.g. 1234
+     * @param metricKeys
+     *                   Comma-separated list of additional fields that can be returned in the response.
+     *                   E.g."new_coverage,new_sqale_debt_ratio,new_uncovered_conditions" */
     @GET()
     @Path("/measures/component")
-    ComponentMeasures getComponentMeasures(@QueryParam("component") String project, @QueryParam("pullRequest") String prId, @QueryParam("metricsKeys") String metricKeys);
+    ComponentMeasures getComponentMeasures(
+            @QueryParam("component") String project,
+            @QueryParam("pullRequest") String prId,
+            @QueryParam("metricKeys") String metricKeys,
+            @QueryParam("additionalFields") @DefaultValue("metrics") String additionalFields);
+
+    default ComponentMeasures getComponentMeasures(String project, String prId, String metricKeys) {
+        return getComponentMeasures(project, prId, metricKeys, "metrics");
+    }
 
+    default IssueSearch getIssuesSearch(String project, String prId, String branch) {
+        return getIssuesSearch(
+                project,
+                prId,
+                branch,
+                "_all",
+                "severities,types",
+                1,
+                500,
+                "false",
+                "FILE_LINE",
+                // INFO got removed here
+                "MINOR,MAJOR,CRITICAL,BLOCKER",
+                "OPEN,REOPENED,CONFIRMED");
+    }
 
-    /**
-     * <a href="https://sonarcloud.io/web_api/api/issues/search">docs</a>
-     * TODO onComponentOnly might be interesting to retrieve global metrics
+    /** <a href="https://sonarcloud.io/web_api/api/issues/search">docs</a> TODO onComponentOnly might be
+     * interesting to retrieve global metrics
      *
-     * @param project          Component key. E.g. my_project
-     * @param prId             Pull request id. Not available in the community edition. E.g. 1234
-     * @param branch           BBranch key. Not available in the community edition. E.g. feature/test
-     * @param additionalFields Comma-separated list of the optional fields to be returned in response.
-     * @param facets           Comma-separated list of the facets to be computed. No facet is computed by default.
-     * @param pageNumber       Page number. E.g. 1
-     * @param pageSize         Page size. Highest number is 500
-     * @param resolved         To match resolved or unresolved issues
-     * @param sort             Sort the issues
-     * @param severities       Comma-separated list of severities
-     * @param statuses         Comma-separated list of statuses
-     */
+     * @param project
+     *                         Component key. E.g. my_project
+     * @param prId
```

Comment:
⚠️ Method has 11 parameters, which is greater than 7 authorized. [details](https://sonarcloud.io/scoding_rules?open=java:S107&rule_key=java:S107)